gss-api-app/src/main/java/com/devoops/controller/auth/AuthUserArgumentResolver.java

```diff
@@ -0,0 +1,42 @@
+package com.devoops.controller.auth;
+
+import com.devoops.service.auth.AuthService;
+import com.devoops.service.auth.jwt.JwtToken;
+import com.devoops.service.auth.jwt.TokenType;
+import com.devoops.service.user.UserService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.core.MethodParameter;
+import org.springframework.http.HttpHeaders;
+import org.springframework.web.bind.support.WebDataBinderFactory;
+import org.springframework.web.context.request.NativeWebRequest;
+import org.springframework.web.method.support.HandlerMethodArgumentResolver;
+import org.springframework.web.method.support.ModelAndViewContainer;
+
+@Slf4j
+@RequiredArgsConstructor
+public class AuthUserArgumentResolver implements HandlerMethodArgumentResolver {
+
+    private final AuthService authService;
+    private final UserService userService;
+
+    @Override
+    public boolean supportsParameter(MethodParameter parameter) {
+        return parameter.hasParameterAnnotation(AuthUser.class);
+    }
+
+    @Override
+    public Object resolveArgument(
+            MethodParameter parameter,
+            ModelAndViewContainer mavContainer,
+            NativeWebRequest webRequest,
+            WebDataBinderFactory binderFactory
+    ) {
+        String accessToken = webRequest.getHeader(HttpHeaders.AUTHORIZATION);
+        if (accessToken == null) {
```

Comment:
```suggestion
        if (StringUtils.isBlank(accessToken)) {
```

헤더 validation 시에, 빈문자열도 같이 검증되어도 괜찮을 것 같아요.

Reply:
[반영 완료]